apis/v1alpha2/object_reference_types.go

```diff
@@ -36,3 +36,33 @@ type LocalObjectReference struct {
 	// +kubebuilder:validation:MaxLength=253
 	Name string `json:"name"`
 }
+
+// ObjectReference identifies an API object including its namespace.
+type ObjectReference struct {
+	// Group is the group of the referent.
+	//
+	// +kubebuilder:validation:MaxLength=253
+	Group string `json:"group"`
+
+	// Kind is kind of the referent.
+	//
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=253
+	Kind string `json:"kind"`
+
+	// Name is the name of the referent.
+	//
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=253
+	Name string `json:"name"`
+
+	// Namespace is the namespace of the backend. When unspecified, the local
```

Comment:
```suggestion
// Namespace is the namespace of the backend. When not specified, the local
```

Reply:
We've used "unspecified" in quite a few places around the API,  any reason for the suggested change?

Reply:
I'll leave this one as is for now.